### DISCUSSION TITLE
tutorials/intro-dgs/09-adding-the-track-type

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#108] [course feedback]
# tutorials/intro-dgs/09-adding-the-track-type

https://www.apollographql.com/tutorials/intro-dgs/09-adding-the-track-type

**Comment by mkatsman** [comment]
method throws an exception:   public void mapTracks(JsonNode tracks) throws IOException {

**Reply by lizhennessy** [reply]
Hi there @mkatsman ! I see what you mean: we have a code block that's missing the `throws IOException` bit! I'll fix that right away. Thank you for pointing it out!
